package.json

```diff
@@ -3,7 +3,8 @@
   "email": "trust2065@gmail.com",
   "license": "UNLICENSED",
   "scripts": {
-    "start": "set NODE_ENV=development&&webpack --mode=development --hot",
+    "start": "set NODE_ENV=production&&webpack --mode=development",
+    "dev": "set NODE_ENV=development&&webpack --mode=development --hot",
```

Comment:
`yarn start` should have everything develop needs. It might not be very useful to make 2 different commands.